README.md

```diff
@@ -176,6 +176,31 @@ stringData:
 type: Opaque
 ```
 
+#### Migrating data from an old AWX instance
```

Comment:
Need to update the table of contents. I use https://github.com/ekalinin/github-markdown-toc